### DISCUSSION TITLE
Problem with getting the benefits of the GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by VitrykushRostyslavAndriyovych** [community/community#133140] [GitHub Education]
### Hi everyone,

I submitted an application and was rejected and asked to update my payment details, I updated them and then immediately received approval, but I did not receive a letter with approval in the mail, I waited 3 days to receive my benefits, but nothing happened.
![Screenshot_1](https://github.com/user-attachments/assets/7bb8dc08-c93d-4a93-866f-87f6b0b2f778)
![Screenshot_3](https://github.com/user-attachments/assets/4654ad48-dd94-4abc-8c7b-eb74351c00df)

**Comment by davevad93** [comment]
Hi @VitrykushRostyslavAndriyovych ,

Student benefits usually take a few days to become available, but sometimes they can take a little longer, even a couple of weeks. The waiting time may vary depending on different factors, such as the volume of applications.

I was notified by email when my benefits became available, but you can also find out by going to the application page. When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @VitrykushRostyslavAndriyovych was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470
